ios/AppLovinMAX.h

```diff
@@ -19,7 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * The primary bridge between JS <-> native code for the AppLovin MAX React Native module.
  */
-@interface AppLovinMAX : RCTEventEmitter<RCTBridgeModule, MAAdDelegate, MARewardedAdDelegate, MAAdViewAdDelegate>
+@interface AppLovinMAX : RCTEventEmitter<RCTBridgeModule, MAAdRevenueDelegate, MAAdDelegate, MARewardedAdDelegate, MAAdViewAdDelegate>
```

Comment:
nit but in our codebase we usually order the protocol declaration from the least specific / most fundamental to most specific
```suggestion
@interface AppLovinMAX : RCTEventEmitter<RCTBridgeModule, MAAdDelegate, MARewardedAdDelegate, MAAdViewAdDelegate, MAAdRevenueDelegate>
```